### DISCUSSION TITLE
Use MLflow to store/monitor model artifacts and params

### DISCUSSION THREAD

**Discussion by svijayaku1-chwy** [mlflow/mlflow#7359] [Q&A] [answered]
Is it possible to use Mlflow for logging/monitoring of model artifacts and parameters but not have the model run in MLflow?
e.g. the model will run and produce artifacts - all of which will be stored in s3. I'd like to upload these artifacts, model file and model params to Mlflow and use it to monitor various model runs.

I came across an example where we can register a saved model pkl file to MLflow model registry and then evaluate the model, but would it be possible to have all of these uploaded separately?

**Comment by JHibbard** [comment, marked as answer]
Hi svijayaku1-chwy,

Yes, this is entirely possible. You can start an experiment and run without saving a model.

```
import mlflow

# You can log everything like normal without the model:
with mlflow.start_run() as run:
    mlflow.log_param("p", 0)
    mlflow.log_artifacts("data", artifact_path="states")
    # etc.
```

Hope that helps!
Jim
